setup.py

```diff
@@ -3,7 +3,7 @@
 setup(
     name='hladny_matfyzak',
 
-    version='0.0.6.1',
+    version='0.0.6.2',
```

Comment:
@tWido I would much prefer `0.0.7` here but it's your call.

My opinion is based on http://semver.org/